concepticondata/README.md

```diff
@@ -1,12 +1,12 @@
 
 # Concepticon Statistics
-* concept sets (used): 3975
+* concept sets (used): 3958
```

Comment:
Does this make sense?

Reply:
Yes. @chrzyki cleaned up old concepts that were duplicates!